hawkular-inventory-parent/hawkular-inventory-api/src/main/java/org/hawkular/inventory/api/InventoryService.java

```diff
@@ -119,4 +119,6 @@
      * @return true is InventoryService is Up and Running
      */
     boolean isRunning();
+
+    Import buildExport();
```

Comment:
Naming types of variables is very a matter of taste.
In Alerting we finished up using plural, Import object perhaps now is weird, "Resources", "Inventory" or something that points to all the data, can be good.

Reply:
Also, I think this should be paginated.
If not a OOME is on the way.

Reply:
I was looking for nice names, +1 for Inventory